Upstream: discussed under http://public.kitware.com/Bug/view.php?id=14601
Sadly the FindFreetype.cmake module appears to be missing a maintainer...

From 3ee48618396d42c9ea5102abf84a26cefe378fcc Mon Sep 17 00:00:00 2001
From: Julian Ospald <hasufell@gentoo.org>
Date: Sun, 8 Dec 2013 13:38:06 +0100
Subject: [PATCH] fix check for freetype-2.5.1

---
 Modules/FindFreetype.cmake | 23 +++++++++++++++++++++--
 1 file changed, 21 insertions(+), 2 deletions(-)

diff --git a/Modules/FindFreetype.cmake b/Modules/FindFreetype.cmake
index ccea991..4796137 100644
--- a/Modules/FindFreetype.cmake
+++ b/Modules/FindFreetype.cmake
@@ -70,6 +70,19 @@ find_path(FREETYPE_INCLUDE_DIR_freetype2 freetype/config/ftheader.h
   PATH_SUFFIXES include/freetype2 include
 )
 
+if(NOT FREETYPE_INCLUDE_DIR_freetype2)
+  find_path(FREETYPE_INCLUDE_DIR_freetype2 config/ftheader.h
+    HINTS
+      ENV FREETYPE_DIR
+    PATHS
+      /usr/X11R6
+      /usr/local/X11R6
+      /usr/local/X11
+      /usr/freeware
+    PATH_SUFFIXES include/freetype2 include
+  )
+endif()
+
 find_library(FREETYPE_LIBRARY
   NAMES freetype libfreetype freetype219
   HINTS
@@ -91,8 +104,14 @@ if(FREETYPE_INCLUDE_DIR_ft2build AND FREETYPE_INCLUDE_DIR_freetype2)
 endif()
 set(FREETYPE_LIBRARIES "${FREETYPE_LIBRARY}")
 
-if(FREETYPE_INCLUDE_DIR_freetype2 AND EXISTS "${FREETYPE_INCLUDE_DIR_freetype2}/freetype/freetype.h")
-    file(STRINGS "${FREETYPE_INCLUDE_DIR_freetype2}/freetype/freetype.h" freetype_version_str
+if(EXISTS "${FREETYPE_INCLUDE_DIR_freetype2}/freetype/freetype.h")
+  set(FREETYPE_H "${FREETYPE_INCLUDE_DIR_freetype2}/freetype/freetype.h")
+elseif(EXISTS "${FREETYPE_INCLUDE_DIR_freetype2}/freetype.h")
+  set(FREETYPE_H "${FREETYPE_INCLUDE_DIR_freetype2}/freetype.h")
+endif()
+
+if(FREETYPE_INCLUDE_DIR_freetype2 AND FREETYPE_H)
+    file(STRINGS "${FREETYPE_H}" freetype_version_str
          REGEX "^#[\t ]*define[\t ]+FREETYPE_(MAJOR|MINOR|PATCH)[\t ]+[0-9]+$")
 
     unset(FREETYPE_VERSION_STRING)
-- 
1.8.5.1

